sample_config/database-layout-update-001.sql

```diff
@@ -0,0 +1,86 @@
+--
+-- Update database layout from previous version
+--
+
+ALTER TABLE timed_thinclient_to_vm_mapping ADD resolution character varying NULL;
+
+DROP VIEW current_thinclient_to_vm_mapping;
+CREATE VIEW current_thinclient_to_vm_mapping AS
+SELECT f.id,
+       f.thinclient,
+       f.vm,
+       f.resolution,
+       f.start_date,
+       f.end_date,
+       f.prio,
+       f.shutdown_vm
+FROM
+  (
+     (SELECT timed_thinclient_to_vm_mapping.id,
+             timed_thinclient_to_vm_mapping.thinclient,
+             timed_thinclient_to_vm_mapping.vm,
+             timed_thinclient_to_vm_mapping.resolution,
+             timed_thinclient_to_vm_mapping.start_date,
+             timed_thinclient_to_vm_mapping.end_date,
+             0 AS prio,
+             timed_thinclient_to_vm_mapping.shutdown_vm
+      FROM timed_thinclient_to_vm_mapping
+      WHERE ((timed_thinclient_to_vm_mapping.start_date IS NULL)
+             AND (timed_thinclient_to_vm_mapping.end_date IS NULL))
+      UNION SELECT timed_thinclient_to_vm_mapping.id,
+                   timed_thinclient_to_vm_mapping.thinclient,
+                   timed_thinclient_to_vm_mapping.vm,
+                   timed_thinclient_to_vm_mapping.resolution,
+                   timed_thinclient_to_vm_mapping.start_date,
+                   timed_thinclient_to_vm_mapping.end_date,
+                   1 AS prio,
+                   timed_thinclient_to_vm_mapping.shutdown_vm
+      FROM timed_thinclient_to_vm_mapping
+      WHERE ((timed_thinclient_to_vm_mapping.start_date <= now())
+             AND (timed_thinclient_to_vm_mapping.end_date IS NULL)))
+   UNION SELECT timed_thinclient_to_vm_mapping.id,
+                timed_thinclient_to_vm_mapping.thinclient,
+                timed_thinclient_to_vm_mapping.vm,
+                timed_thinclient_to_vm_mapping.resolution,
+                timed_thinclient_to_vm_mapping.start_date,
+                timed_thinclient_to_vm_mapping.end_date,
+                2 AS prio,
+                timed_thinclient_to_vm_mapping.shutdown_vm
+   FROM timed_thinclient_to_vm_mapping
+   WHERE ((now() >= timed_thinclient_to_vm_mapping.start_date)
+          AND (now() <= timed_thinclient_to_vm_mapping.end_date))) f
+ORDER BY f.prio DESC,
+         f.start_date,
+         f.id;
+
```

Comment:
Can you clean up the indentation and code layout a bit, and maybe add a few comments? See our SQL style guides for examples. This would help understanding this long VIEW